api/tests/Feature/PoolCandidateTest.php

```diff
@@ -30,14 +30,15 @@ public function testFilterByClassification(): void
     Classification::factory()->count(3)->create();
     PoolCandidate::factory()->count(5)->create([
       'expected_salary' => [], // remove salaries to avoid accidental classification-to-salary matching
+      'expiry_date' => '2050-01-01', // ensure no candidates are expired for this test
```

Comment:
This is a "magic number" which is an anti-pattern.  Try making a constant for it with a descriptive name like FAR_FUTURE_DATE and use that through the file instead.